src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md

```diff
@@ -0,0 +1,50 @@
+---
+group: frontend-developer-guide
+title: Custom validation rules
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+Custom validation rules can be added by creating a Javascript mixin for the `mage/validation` module and calling the `$.validator.addMethod` function with the custom validation rule parameters as described below:
+
+```javascript
+$.validator.addMethod(
+  'rule-name',
+  function(value, element) {
+    // Return true or false after validation rule check
+  },
+  $.mage.__('Error message to display if validation fails')
+)
+```
+
+For example, this code snippet adds a new simple validation rule in the mixin to validate if an input field has only five words.
+
+`requirejs-config.js`
```

Comment:
Could I ask you to provide a relative path to the file, please? It will make the process easier to understand for new developers